### ISSUE TITLE
How long did the training take? 

### ISSUE THREAD

**Issue by umitkacar** [yaojieliu/CVPR2019-DeepTreeLearningForZeroShotFaceAntispoofing#14]
1- How long did the training take? 
2- Which GPU card did you use? 
3- Have you trained as follows?

config.STEPS_PER_EPOCH = 1000
config.MAX_EPOCH = 40
config.BATCH_SIZE = 32

4- What are the final loss function values after training ends?

Map loss = 0.001, ???
Cls loss = 0.001, ???

Thanks in advanced
